app/src/main/java/me/hufman/androidautoidrive/music/spotify/SpotifyWebApi.kt

```diff
@@ -72,6 +76,79 @@ class SpotifyWebApi private constructor(val context: Context, val appSettings: M
 		spotifyAppControllerCaller = null
 	}
 
+	/**
+	 * Creates a private playlist with the provided name. The newly created playlist's [PlaylistUri]
+	 * is returned.
+	 */
+	suspend fun createPlaylist(playlistName: String): PlaylistUri? {
```

Comment:
Should the description be passed in as well, allowing for different descriptions for different playlists in the future?

Reply:
Yeah it should be to ensure greater flexibility for future expansion, updated this method signature to take in a playlist description parameter that defaults to null